src/rgw/rgw_common.cc

```diff
@@ -513,7 +513,7 @@ bool parse_rfc2616(const char *s, struct tm *t)
 bool parse_iso8601(const char *s, struct tm *t, uint32_t *pns, bool extended_format)
 {
   memset(t, 0, sizeof(*t));
-  const char *p;
+  const char *p = nullptr;
```

Comment:
@joscollin: yeah, I have similar feelings regarding false positive. I'm not against merge if the change helps to minimize the scan's noise. Though, I would appreciate putting a comment in such situation. Alternatively, can we blacklist the item to not pollute further runs?